msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java

```diff
@@ -49,6 +50,25 @@
 
     private static final String TAG = BrokerClientIdRefreshTokenAccessor.class.getSimpleName();
 
+    private static void throwIfNotValidBroker(final Context context) throws MsalClientException {
```

Comment:
>private static void throwIfNotValidBroker [](start = 4, length = 41)

nit: Should this be moved down along with the other private method?

Reply:
Addressed